<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WIN-39319556043" failures="1" tests="6" name="Message Classification Change Set" time="29.411" errors="0" timestamp="2022-07-26T16:27:48 CST">
  <testcase classname="com.rs.test.messageClassification.ChangeSet.ChangeSet" name="createChangeSet" time="18.917">
    <failure type="java.lang.AssertionError" message="The record was not created successfully expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The record was not created successfully expected [true] but found [false]
at com.rs.test.pageObjectModel.messageClassification.ChangeSet.ChangeSetPO.createChangeSet(ChangeSetPO.java:476)
at com.rs.test.messageClassification.ChangeSet.ChangeSet.createChangeSet(ChangeSet.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- createChangeSet -->
  <testcase classname="com.rs.test.messageClassification.ChangeSet.ChangeSet" name="editChangeSet" time="0.0">
    <skipped/>
  </testcase> <!-- editChangeSet -->
  <testcase classname="com.rs.test.messageClassification.ChangeSet.ChangeSet" name="createChangeSetMessageClassification" time="0.0">
    <skipped/>
  </testcase> <!-- createChangeSetMessageClassification -->
  <testcase classname="com.rs.test.messageClassification.ChangeSet.ChangeSet" name="editChangeSetMessageClassification" time="0.0">
    <skipped/>
  </testcase> <!-- editChangeSetMessageClassification -->
  <testcase classname="com.rs.test.messageClassification.ChangeSet.ChangeSet" name="deleteChangeSetMessageClassification" time="0.0">
    <skipped/>
  </testcase> <!-- deleteChangeSetMessageClassification -->
  <testcase classname="com.rs.test.messageClassification.ChangeSet.ChangeSet" name="deleteChangeSet" time="0.0">
    <skipped/>
  </testcase> <!-- deleteChangeSet -->
</testsuite> <!-- Message Classification Change Set -->
